### ISSUE TITLE
Use a second vertex buffer for instances

### ISSUE THREAD

**Issue by expenses** [sotrh/learn-wgpu#116]
Assuming you have a second [`VertexBufferDescriptor`](https://docs.rs/wgpu/0.6.0/wgpu/struct.VertexBufferDescriptor.html) set with [`InputStepMode::Instance`](https://docs.rs/wgpu/0.6.0/wgpu/enum.InputStepMode.html#variant.Instance), you can simply set an instance buffer like so:
```rust
render_pass.set_vertex_buffer(1, &self.instance_buffer.slice(..));
```

This is a lot easier than adding it to the bind group like you're doing in the instancing tutorial.

**Comment by sotrh** [comment]
You are correct. I originally showcased multiple methods of instancing, including using a texture. I used storage buffers as the default method, as I was interested in them at the time.

I'm thinking of switching to using vertex buffers for instancing, but I'm currently busy getting `wgpu` to work with `wasm-pack`. You're welcome to make a pull request.

**Comment by sotrh** [comment]
Fixed in #126
